lib/standards/index.js

```diff
@@ -0,0 +1,31 @@
+import ariaAttrs from './aria-attrs';
+import { clone } from '../core/utils';
+
+const origAriaAttrs = clone(ariaAttrs);
+const standards = {
+	ariaAttrs
+};
+
+// @see https://stackoverflow.com/a/59008477/2124254
+function merge(current, updates) {
+	for (const key of Object.keys(updates)) {
+		if (!current.hasOwnProperty(key) || typeof updates[key] !== 'object') {
```

Comment:
We don't want to merge arrays. That'd get confusing. Stuff like:

```js
merge(['foo', 'bar'], ['bar']) // returns ['bar', 'bar'], but should return ['bar']
```

```suggestion
		if (!current.hasOwnProperty(key) || typeof updates[key] !== 'object' || Array.isArray(current[key])) {
```